src/libstd/net/parser.rs

```diff
@@ -388,10 +388,11 @@ impl FromStr for SocketAddr {
 /// use std::net::SocketAddr;
 ///
 /// // No problem, the `panic!` message has disappeared.
-/// let _foo: SocketAddr = "127.0.0.1:8080".parse().unwrap();
+/// let _foo: SocketAddr = "127.0.0.1:8080".parse().expect("unreachable panic");
 /// ```
 ///
 /// [`FromStr`]: ../../std/str/trait.FromStr.html
+/// [`AddrParseError`]: ../../std/net/struct.AddrParseError.html
```

Comment:
Just like you said, it's useless to do this since we're already on the correct page (sorry again about this...).

Reply:
Hi,

Yes, it was removed. See 11be095203a084733966e9d52b1c0489bbd22056.